doc-content/enterprise-only/installation/eap-ba-dm-run-proc.adoc

```diff
@@ -5,6 +5,9 @@ Use this procedure to run the {PRODUCT} on {EAP} in standalone mode.
 .Prerequisites
 * {PRODUCT} is installed and configured.
 
+NOTE: If you changed the default host (`localhost`) or the default port (`9990`), you must edit the `business-central.war/WEB-INF/classes/datasource-management.properties`
+and `business-central.war/WEB-INF/classes/security-management.properties` files as described in https://access.redhat.com/solutions/3519551[Solution 3519551] before you run {PRODUCT}.
+
```

Comment:
Suggest "If you changed ..., then before you run {PRODUCT}, you must edit the ..." It's clearest imo to begin with when something is done, then what is done. "After you X, ..."  "Before you X, ..."  Some people argue that the syntax should just mirror the order in which you actually do the thing, so "After you X, ..." but "Do W before X." But this doesn't work in long constructs like this one, where the "before" bit is way down the road, plus even in short cases, I still believe it's clearest to state the temporal constraint and then the action. TMI, I know, but just so you don't think it's entirely just subjective.